src/Pages/home/Containers/HomeContainers.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  TableBody,
+  makeStyles,
+  Paper,
+  TableRow,
+  Button,
+} from "@material-ui/core";
+import React from "react";
+import SearchTables from "../Components/SearchTable";
+import Tables, { createData } from "../Components/Tables";
+
+const useStyles = makeStyles((theme) => ({
+  pageContent: {
+    margin: theme.spacing(5),
+    padding: theme.spacing(3),
+    borderRadius: "15px",
+  },
+  buttonView: {
+    backgroundColor: "transparent",
+    color: "#CF658D",
+    border: "1.5px solid #CF658D",
+    textTransform: "none",
+    borderRadius: "20px",
+    fontSize: "16px",
+    height: "30px",
+  },
+  root: {
+    "&:nth-of-type(odd)": {
+      backgroundColor: "#FFF6FD",
+    },
+  },
+}));
+
+const headCells = [
+  { id: "testID", label: "Test ID" },
+  { id: "date", label: "Date" },
+  { id: "patientID", label: "Patient ID" },
+  { id: "name", label: "Name" },
+  { id: "age", label: "Age" },
+  {
+    id: "cardioembolicProbability",
+    label: "Cardioembolic Probability (%)",
+  },
+];
+
+const rows = [
+  createData("1150", "28/02/2564", "12345678", "Somachi Tang", 23, 0.23),
+  createData("1134", "19/02/2564", "12435637", "Evan Jefferson", 31, 0.55),
+  createData("1119", "06/02/2564", "10003484", "Nattasuk Chaithana", 38, 0.12),
+  createData("1118", "06/02/2564", "10475995", "Srisawut Somkierd", 42, 0.23),
+  createData("1107", "04/02/2564", "20384857", "Meethana Julsiri", 26, 13),
+  createData("1092", "25/01/2564", "20038480", "Sanphop Klinpipat", 63, 18),
+  createData("1088", "25/01/2564", "10004832", "Malini Smith", 49, 22),
+  createData("1056", "16/01/2564", "10003884", "Evan Jeffer", 68, 27),
+  createData("1045", "15/01/2564", "17399400", "Leelerd Somjai", 50, 19),
+  createData("1001", "12/01/2564", "19293948", "Srisaket Charoensri", 37, 25),
+];
+
+const HomeContainer: React.FC = () => {
+  const classes = useStyles();
+  const { TblContainer, TblHead, TblCenter, TblPagination } = Tables({
+    rows: rows,
+    headCells: headCells,
+  });
+
+  return (
+    <>
+      <SearchTables title="Patient Tests " subTitle="Total 63 records " />
```

Comment:
Remove last space if it hasn't necessary.

Reply:
Remove space
<img width="172" alt="ภาพถ่ายหน้าจอ 2564-07-18 เวลา 10 56 15" src="https://user-images.githubusercontent.com/59549553/126055168-5fa6c6bb-18d1-4903-8ac3-d12faddffdc4.png">

Reply:
> Remove space
> <img alt="ภาพถ่ายหน้าจอ 2564-07-18 เวลา 10 56 15" width="172" src="https://user-images.githubusercontent.com/59549553/126055168-5fa6c6bb-18d1-4903-8ac3-d12faddffdc4.png">

I mean space of props hahaha.
![image](https://user-images.githubusercontent.com/54408097/126055332-ac9b6ece-1bc5-403d-9f8e-34b0b0999879.png)